index.bs

```diff
@@ -179,29 +186,33 @@ run the following steps:
 
 1. If <a>Picture-in-Picture support</a> is `false`, throw a
     {{NotSupportedError}} and abort these steps.
-2. If the document is not <a>allowed to use</a> the <a>policy-controlled feature</a>
+2. Let |document| be the <a>current settings object</a>'s <a>relevant global object</a>'s
```

Comment:
Then you can revert this change so that your PR affects only the last part of this algorithm

Reply:
I can but you want that change in another PR then? Otherwise it is not clear where document comes from

Reply:
Done!